server.js

```diff
@@ -19,7 +19,7 @@ require("./config/passport")(passport);
 connectDB();
 
 app.set("view engine", "ejs");
-app.use(express.static("public"));
+app.use('/public', express.static("public"));
```

Comment:
@GrantOkawa I'll approve this for merge if we revert line 22 to `app.use(express.static("public"));`

make the change, commit it, and push it.